graphs/bipartite.py

```diff
@@ -0,0 +1,52 @@
+# Uses python3
+import sys
+
+
+def check(partition, adj):
```

Comment:
Type hints and doctests as discussed in CONTRIBUTING.md?  Could we have a more self documenting function name?  What exactly are we checking?

Reply:
updated the function name